binder/postBuild

```diff
@@ -37,11 +37,18 @@ jupyter labextension link --no-build packages/application/
 jupyter labextension link --no-build packages/ui-components/
 jupyter labextension link --no-build packages/metadata-common/
 jupyter lab build --dev-build=False --minimize=False --debug
+
+pwd
+
 mkdir -p binder-demo
 
 # Enable debug output
 set -x
 
+# Hack option 2: "hide" the fact that the root directory is a cloned github repository
```

Comment:
"Hack option 2:" doesn't really have any context here for folks looking at this code.  Perhaps adding a link to the applicable issue or removing this preamble would be more helpful?

Reply:
You are right. I changed the comment to read:

_repo2docker clones the Elyra repository before the Docker image is build and uses the cloned directory as "root". The git extension recognizes that the directory is git-enabled and therefore prevents the user from cloning other repositories using the UI. Remove the git artifacts in the root directory to work around this issue._